engine/api/sdk-index-new.md

```diff
@@ -0,0 +1,189 @@
+---
+
+title: Develop with Docker Engine SDKs description: Using Docker SDKs to automate Docker tasks in your language of choice keywords: developing, sdk redirect_from:
+
+-	/engine/api/sdks/
+-	/develop/sdk/ ---
+
+Docker provides an API for interacting with the Docker daemon (called the Docker Engine API), as well as SDKs for Go and Python. The SDKs allow you to build and scale Docker apps and solutions quickly and easily. If Go or Python don't work for you, you can use the Docker Engine API directly.
+
+The Docker Engine API is a RESTful API accessed by an HTTP client such as `wget` or `curl`, or the HTTP library which is part of most modern programming languages.
+
+Install the SDKs
+----------------
+
+Use the following commands to install the Go or Python SDK. Both SDKs can be installed and coexist together.
+
+### Go SDK
+
+> **Important**: the Go SDK is versioned by Docker release. Docker releases use a YY.MM versioning scheme, which does not follow SemVer semantics. When using Go modules, `go mod` by default looks for tags matching the SemVer format, which results in it to download an obsolete version of the Docker SDK (v1.13.1). When initializing your project, instruct `go mod` to use the current version of the SDK.
+
+The example below initializes a new project to use the `v19.03.6` version of the Docker SDK:
+
+```bash
+mkdir myproject
+cd myproject
+
+# Initialize go modules
+go mod init myproject
+
+# Add Docker SDK v19.03.6 module to go.mod
+go get github.com/docker/docker/client@v19.03.6
+```
+
+The client requires a recent version of Go. Run `go version` and ensure that you are running a currently supported version of Go
+
+[Read the full Docker Engine Go SDK reference](https://godoc.org/github.com/docker/docker/client).
+
+### Python SDK
+
+-	**Recommended**: Run `pip install docker`.
+
+-	**If you can't use `pip`**:
+
+	1.	[Download the package directly](https://pypi.python.org/pypi/docker/).
```

Comment:
Is this a formatting issue?

Reply:
oh! boo! this was a scratch file I had on my machine, shouldn't have committed it in this PR; let me remove it 😅. Thanks for catching!

Reply:
Fixed 👍